stun_server/region/dns.tf

```diff
@@ -0,0 +1,11 @@
+data "cloudflare_zone" "dns_zone" {
+  name = var.domain_name
+}
+
+resource "cloudflare_record" "instance_dns" {
+  zone_id = data.cloudflare_zone.dns_zone.id
+  name    = join("-", ["stun", data.aws_region.current.name])
+  content = data.aws_network_interface.stun_server_interface.association[0].public_ip
```

Comment:
How does this handle rolling updates?
How does this work with new deployments?
stun_server_interface does a filter, right now, that will return nothing?
During a rolling update it will return old values?
Do we need to adjust the dependency here to ensure this resource is updated after the new task is active?

Reply:
This needs to be tested after the merge. In theory, it should filter after ECS is deployed due to this config:
```
# Required to fetch the public IP address of the ECS service
  enable_ecs_managed_tags = true
  wait_for_steady_state   = true
```

Reply:
Even with `depends_on` being blank?

Reply:
Added `depends_on` as we discussed.